google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -231,7 +238,12 @@ private void createUIComponents() {
                 CloudLibrary library =
                     (CloudLibrary)
                         cloudLibrariesTable.getModel().getValueAt(selectedIndex, CLOUD_LIBRARY_COL);
-                detailsPanel.setCloudLibrary(library, apiManagementMap.get(library));
+                detailsPanel.setCloudLibrary(
```

Comment:
Nit: pretty long event handling call here as part of component initialization, maybe it could read better as a separate event handler method?

Reply:
done